.travis.yml

```diff
@@ -28,17 +25,14 @@ before_install:
     fi
 
 install:
-  - nvm install $TARGET_NODE_VERSION
-  - nvm use $TARGET_NODE_VERSION
-  - node --version
   - npm --version
   - npm install -g elm@$ELM_VERSION
   - mv $(npm config get prefix)/bin/elm $(npm config get prefix)/bin/elm-old
   - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
   - chmod +x $(npm config get prefix)/bin/elm
   - npm install
-  - pushd tests && elm make --yes && popd
+  - pushd tests && elm make && popd
```

Comment:
I think this should not be part of install. install gets cached this shouldn't

Reply:
What should not be part of install? `elm-make`?

Reply:
yes, formally it is not an install, it is a build, move it to `script` part ;)